models/articles-model.js

```diff
@@ -18,6 +19,32 @@ function retrieveArticles(article_id) {
     })
 }
 
+function newComment(article_id, article_body) {
+    
+    const criteria = ['author', 'body']
+    if (!criteria.every(objCriteria => Object.keys(article_body).includes(objCriteria))) {
```

Comment:
This if statement seems a bit bloated... if you're checking that it has a `body` and `author` key can you use `!article_body.body` and same for `.author` ?